.github/workflows/deploy.yml

```
@@ -1,46 +1,31 @@
-      # Setup gcloud CLI
-      - name: Deploy (🚀)
-        uses: google-github-actions/deploy-cloudrun@v1
-        with:
-          service: clima
-          image: us-docker.pkg.dev/clima-316917/gcr.io/clima
-          region: us-central1
-          credentials: ${{ secrets.GCP_SA_KEY_JSON }}
-          project_id: clima-316917
\ No newline at end of file
+          --project=clima-316917 \
+          --substitutions=_REPO_NAME="clima",_PROJ_NAME="clima-316917",_IMG_NAME="main",_GCR="us.gcr.io",_REGION="us-central1",_MEMORY="4Gi",_CPU="2"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)